modules/conversion-center/assets/scss/widgets/contact-buttons-var-4.scss

```diff
@@ -84,41 +85,80 @@
 		margin-bottom: var(--e-contact-buttons-contact-gap);
 	}
 
-	.e-contact-buttons__contact-icon-link {
+	.e-contact-buttons__contact-icon-container {
 		align-items: center;
 		background-color: var(--e-contact-buttons-contact-button-bg);
 		border-radius: 50px;
 		display: inline-flex;
 		justify-content: center;
 		transition: $transition-hover;
+	}
+
+	.e-contact-buttons__contact-icon-link {
+		align-items: center;
+		display: flex;
+		flex-direction: row;
+		gap: 14px;
 
 		svg {
 			fill: var(--e-contact-buttons-contact-button-icon);
 		}
 
 		&:hover,
 		&:focus {
-			background-color: var(--e-contact-buttons-contact-button-bg-hover);
-			transition: $transition-hover;
+
+			.e-contact-buttons__contact-icon-container {
+				background-color: var(--e-contact-buttons-contact-button-bg-hover);
+				transition: $transition-hover;
+			}
 
 			svg {
 				fill: var(--e-contact-buttons-contact-button-icon-hover);
 			}
 		}
 
-		&.has-size-small {
+		&.has-size-small .e-contact-buttons__contact-icon-container {
 			height: var(--e-contact-buttons-icon-size-small);
 			width: var(--e-contact-buttons-icon-size-small);
 		}
 
-		&.has-size-medium {
+		&.has-size-medium .e-contact-buttons__contact-icon-container {
 			height: var(--e-contact-buttons-icon-size-medium);
 			width: var(--e-contact-buttons-icon-size-medium);
 		}
 
-		&.has-size-large {
+		&.has-size-large .e-contact-buttons__contact-icon-container {
 			height: var(--e-contact-buttons-icon-size-large);
 			width: var(--e-contact-buttons-icon-size-large);
 		}
 	}
+
+	.e-contact-buttons__contact-tooltip {
+		background-color: var(--e-contact-buttons-tooltip-bg);
+		border-radius: 16px;
+		box-shadow: 2px 4px 4px 0 rgba(0, 0, 0, 0.15);
+		color: var(--e-contact-buttons-tooltip-text);
+		font-size: 13px;
+		line-height: 25px;
+		padding: 4px 14px;
+		position: relative;
+
+		&::after {
+			border-color: transparent transparent transparent var(--e-contact-buttons-tooltip-bg);
+			border-style: solid;
+			border-width: 8px 0 8px 10px;
```

Comment:
Maybe replace with logical properties without the shorthand, like: 
border-inline-start-width
border-inline-end-width
border-block-start-width
border-block-end-width

Reply:
Thanks!